content/en/docs/faq/multi-cluster-management/host-cluster-access-member-cluster.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Restore the Host Cluster Access to A Member Cluster"
+keywords: "Kubernetes, KubeSphere, Multi-cluster, Host Cluster, Member Cluster"
+description: "How to restore the Host Cluster access to a Member Cluster"
+linkTitle: "Restore the Host Cluster Access to A Member Cluster"
+Weight: 16710
+---
+
+KubeSphere features [multi-cluster maganement](../../../multicluster-management/introduction/kubefed-in-kubesphere/) and tenants with necessary permissions (usually cluster administrators) can access the central control plane from the Host Cluster to manage all the Member Clusters. It is highly recommended that you manage your resources across your cluster through the Host Cluster.
+
+This tutorial demomstrates how to restore the Host Cluster access to a Member Cluster.
+
+## Possible Error Message
+
+If you can't access a Member Cluster from the central control plane and your browser keeps redirecting you to the login page of KubeSphere, run the following command on that Member Cluster to get the logs of the ks-apiserver.
+
+```
+kubectl -n kubesphere-system logs ks-apiserver-7c9c9456bd-qv6bs
+```
+
+{{< notice note >}}
+
+`ks-apiserver-7c9c9456bd-qv6bs` refers to the pod ID on that Member Cluster. Make sure you use the ID of your own pod.
```

Comment:
refers to the pod ID on that Member Cluster. Make sure you use the ID of your own pod.
->
refers to the Pod ID on that Member Cluster. Make sure you use the ID of your own Pod.